### ISSUE TITLE
[Bug]: Opening video in new window now refreshes the Subscriptions page

### ISSUE THREAD

**Issue by gtg250y** [FreeTubeApp/FreeTube#2458]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.

### Describe the bug

1. From the Subscriptions page (in any window), middle-click a video *that you haven't opened before*, to open it in a new window.
2. That reloads any/all windows that were showing Subscriptions page, scrolled all the way back to the top.

### Expected Behavior

- Launching a video in a new window from the Subscriptions page shouldn't affect that prior window, or at least it shouldn't lose track of the scroll position.

### Issue Labels

usability issue

### FreeTube Version

v0.17.0

### Operating System Version

Windows 8

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.16.0

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
i am unable to reproduce this. Could u provide a screenrecording of this?

**Comment by gtg250y** [comment]
It's more inconsistent that I realized, and I can't figure out any perfect way to replicate. I thought I fixed it by deleting all subscriptions (was up to 99) and re-adding many of them. But I can still have it happen (seems rarer now) where the subscriptions page refreshes on its own when something gets launched in another window (could be a video, or even launching open a channel page in a new window).
